cost/aws/rightsize_ebs_volumes/aws_volumes_rightsizing.pt

```diff
@@ -503,10 +600,12 @@ script "js_combine_cloudwatch_data", type: "javascript" do
       "volumeType": vol_read_data.volumeType,
       "iops": vol_read_data.iops,
       "throughput": vol_read_data.throughput,
-      "iopsAverage": iops_average,
+      "iopsAverage": Math.round(iops_average*100)/100,
```

Comment:
How does this work? Shouldn't it be: Math.round(ios*100)/100, ?

It looks like "iopsAverage" value is rounding itself without existing before

Reply:
In which line is this value rounded?

Reply:
I mean, you're using Math.round(iops_average*100)/100, but what is the value of iopsAverage?